source/draw/Image.ooc

```diff
@@ -66,6 +71,9 @@ Image: abstract class {
 		this resizeTo(((this size toFloatSize2D()) * Float minimum(restriction width as Float / this size width as Float, restriction height as Float / this size height as Float)) toIntSize2D())
 	}
 	resizeTo: abstract func (size: IntSize2D) -> This
+	resizeTo: virtual func ~withMethod (size: IntSize2D, method: TransformMethod) -> This {
+		this resizeTo(size)
```

Comment:
Yes, it is meant to be reimplemented in derived classes. If a derived class *knows* how to resize itself according to given method, then it should reimplemented this function. Otherwise it just defaults to standard `resizeTo`.